src/main/java/com/Acrobot/Breeze/Utils/InventoryUtil.java

```diff
@@ -41,10 +41,6 @@ public static int getAmount(ItemStack item, Inventory inventory) {
             return 0;
         }
 
-        if (inventory.getType() == null) {
-            return Integer.MAX_VALUE;
-        }
-
```

Comment:
`inventory.getType()` can never be null in more recent versions, so this check would be obsolete.
Now knowing you want backwards compatibility to 1.13 I'll revert this change aswell.

From the JavaDocs:
```
@NotNull
InventoryType getType()
```
Now knowing you want backwards compatibility to 1.13 it would actually be required for the older versions.
I'll revert this change aswell.